### DISCUSSION TITLE
How to handle transfers with the importer

### DISCUSSION THREAD

**Discussion by bernhardh** [firefly-iii/firefly-iii#4447] [Firefly III]
I am running Firefly III version 5.4.6

**Description**

I have two accounts (A and B) in firefly and for both I am using the CSV importer and it works.

But I don't understand how to handle transfers between these two accounts? Let's say I transfer 100€ from A to B over my online banking and then I run the csv importer for both accounts. The importer imports 100€ as a expense transaction for A and 100€ as a revenue transaction for account B. 

Thats not only incorrect, but there is also a problem on the dashboard with that. I don't know exactly why, but this leads to a negative balance on the dashboard, even if both accounts have a positive balance.

**What I tried**

Now I tried to work with rules. I created a rule for account A to change these transactions to a transfer. That works. But know I have too much mony on account B because there is now a 100€ transaction plus a 100€ revenue. Now I added an extra rule for Account B which deletes the revenue transactions and now I have a negative balance which isn't true as well.

How to handle these situations?

Thank you very much for this great product!

**Comment by JC5** [comment]
I put something in the docs today because this question comes up often. Let me know if this clears things up:

https://docs.firefly-iii.org/csv/usage/importing_transfers/

**Comment by bernhardh** [comment]
Hello,

thanks for responding. I have now tried your suggestion and it creates the transfer. But now the problem is, that it creates the transfer twice, which sound "logical" for me in this setup, because in both exports there is a line with 100€. One time with a plus and the other with a minus.

**Reply by JC5** [reply]
Yes, and I expect some fields are different, like the description or some meta data.
